src/test/java/BaseTest.java

```diff
@@ -0,0 +1,82 @@
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.builder.ResponseSpecBuilder;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import io.restassured.specification.ResponseSpecification;
+import org.example.pojoClasses.BoardResponse;
+import org.example.utility.AllureLogger;
+import org.example.utility.FrameworkUtility;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+
+import static io.restassured.RestAssured.given;
+
+
+public abstract class BaseTest extends FrameworkUtility {
+
+    protected static RequestSpecification requestSpec;
+    protected static ResponseSpecification responseSpec;
+    private static final String CONTENT_TYPE = "Content-Type";
+    private static final String APPLICATION_JSON = "application/json";
+
+    protected Response getRequest(String endPoint, String boardId) {
```

Comment:
Usually we store such methods in src/main/.../dao (Data Access Objects) package, each class can represent group of endpoints.